### DISCUSSION TITLE
[Feature] RAW images icon indicator

### DISCUSSION THREAD

**Discussion by ArtemSmaznov** [immich-app/immich#14852] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, the only way to tell a RAW photo apart from it's traditional jpg/png/etc. counterparts is to view the extension of the file in photos' info under details. In the timeline view you can sort of tell them apart on PC as there look slightly different color-wise, but it is sometimes hard to tell which one is which. And the pair of pictures taken don't always appear in the same order. They look identical on my android, probably because it has an HDR screen.

This makes it very inconvenient to quickly select a non-RAW version to share or identify which image needs to be selected first for stacking so that RAW one doesn't appear on the top of the stack.

This is a request to add a generic RAW icon to all raw image thumbnails (maybe even to preview view as well?)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6267

**Reply by ArtemSmaznov** [reply]
Thanks, I didn't have a thorough look through the closed issues.

I would like to start a discussion again though. Here is my typical flow from my phone and I don't see how searching could simplify it:

1. take N pictures
2. open immich
3. 2 new identical thumbnails appear in the timeline for each photo taken
4. tap on the last picture in the timeline to open it
5. swipe up to open picture info
6. check file extension under details
7. swipe down to close picture info
8. press back to quit back to timeline
9. of the 2 identical images select the one that is not RAW based on step 6
10. then select the other one
11. tap the stack button in the toolbar
12. tap the picture to pen it and select share or tap-hold and select share
13. proceed sharing as needed

I would love to eliminate steps 4-8 + decision making of step 9 and a RAW icon on just one of the photos, which is really not meant not be seen unless you primary work with RAWs (and I would argue this is a somewhat niche use case in and of itself). #8889 would probably fix this, but still, I see no harm in having a RAW icon. Sort of like a "this is probably not what you want" kind of icon. And the compressed ones remain clean. I can easily see RAW icon sitting in place of a stack one and visually standing out.
